test/built-ins/String/prototype/padEnd/exception-not-object-coercible.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2016 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-string.prototype.padend
+description: String#padEnd should fail if given a null or undefined value
+author: Jordan Harband
+---*/
+
+assert.throws(TypeError, function () {
+    String.prototype.padEnd.call(null);
+});
```

Comment:
you may store `String.prototype.padEnd` on a variable outside of this function so we avoid false positives when `padEnd` is still `undefined`.